e2e/test/scenarios/embedding-sdk/static-dashboard.cy.spec.js

```diff
@@ -0,0 +1,82 @@
+import {
+  ORDERS_DASHBOARD_DASHCARD_ID,
+  ORDERS_QUESTION_ID,
+} from "e2e/support/cypress_sample_instance_data";
+import {
+  getTextCardDetails,
+  restore,
+  setTokenFeatures,
+  visitFullAppEmbeddingUrl,
+} from "e2e/support/helpers";
+import {
+  describeSDK,
+  EMBEDDING_SDK_STORY_HOST,
+} from "e2e/support/helpers/e2e-embedding-sdk-helpers";
+import {
+  JWT_SHARED_SECRET,
+  setupJwt,
+} from "e2e/support/helpers/e2e-jwt-helpers";
+
+describeSDK("scenarios > embedding-sdk > static-dashboard", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+    setTokenFeatures("all");
+    setupJwt();
+
+    const textCard = getTextCardDetails({ col: 16 });
+    const questionCard = {
+      id: ORDERS_DASHBOARD_DASHCARD_ID,
+      card_id: ORDERS_QUESTION_ID,
+      row: 0,
+      col: 0,
+      size_x: 16,
+      size_y: 8,
+      visualization_settings: {},
+    };
+
+    cy.createDashboard(
+      {
+        name: "Embedding Sdk Test Dashboard",
+        dashcards: [questionCard, textCard],
+      },
+      { wrapId: true },
+    );
+
+    cy.signOut();
+
+    cy.intercept("GET", "/api/dashboard/*").as("getDashboard");
+    cy.intercept("GET", "/api/user/current").as("getUser");
+  });
+
+  it("should show dashboard content", () => {
+    cy.get("@dashboardId").then(dashboardId => {
+      visitFullAppEmbeddingUrl({
+        url: EMBEDDING_SDK_STORY_HOST,
+        qs: { id: "embeddingsdk-staticdashboard--default", viewMode: "story" },
+        onBeforeLoad: window => {
+          window.JWT_SHARED_SECRET = JWT_SHARED_SECRET;
+          window.METABASE_INSTANCE_URL = "http://localhost:4000";
```

Comment:
Please do not hard code Cypress url.

```suggestion
          window.METABASE_INSTANCE_URL = Cypress.config().baseUrl;
```